### PR TITLE
Create manage view postings

### PR DESCRIPTION
**Pull request by ramy-att**
- Allow the employer to create, edit and view postings (frontend portion) (#87) (#63)
- Created error page (#89)
- Saved user data in an app-wide context and redirect moved to App.js
- And some Navbar links fixes - related to #69 

We need to make the connection to the backend to fetch live postings, and applications.
@khaled106 @mario-elshaer Please carefully review the code and note any errors and/or questions.
Thanks

### CODE REVIEW

File: client/src/components/NavBar/OurNav.js [hunk 1]
```diff
@@ -5,48 +5,62 @@ import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import { NavLink } from "react-router-dom";
 import "./OurNav.css";
+import { useContext } from "react";
+import { UserContext } from "../../App";
```
Comment (khaled106) [inline client/src/components/NavBar/OurNav.js:9]: Nice Use of the function it makes our code cleaner!!
Good work Ramy!!

**Comment by khaled106** [comment]
@ramy-att  The code seems fine to me I just need to run it on my end to confirm a couple of things, but whenever I try to connect the backend it says it can't connect to the database
So once I run it on my end I will approve then we can merge!!

**Review by khaled106 — DISMISSED** [review]
All looks good from the employer side once we do the connection between the backend and front I believe all would be good!!
I just noticed some changes need to be done on the student side, but since the PR mainly for the employers ,so we all good.
Great Work Ramy!! Thank You!!

File: client/src/components/NavBar/OurNav.js (hunk 1 above)
Comment (ramy-att) [reply client/src/components/NavBar/OurNav.js:9]: Thanks!

**Review by mario-elshaer — APPROVED** [review]
I have added some code regarding the footer and nav bar.

**Pull request merged**